src/Microsoft.DotNet.Build.Tasks/PackageFiles/executor/executor.cs

```diff
@@ -0,0 +1,227 @@
+﻿using System;
+using System.IO;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System.Collections.Generic;
+
+namespace Executor
+{
+    public class Executor
+    {
+        public Setup openFile(string filePath)
+        {
+            string jsonFile = string.Empty;
+            if (File.Exists(filePath))
+            {
+                jsonFile = File.ReadAllText(filePath);
+                Setup jsonSetup = JsonConvert.DeserializeObject<Setup>(jsonFile);
+                return jsonSetup;
+            }
+            return null;
+        }
+
+        public void usage()
+        {
+            Console.WriteLine("Executor tool");
+            Console.WriteLine("Usage: executor [config file] [command]");
+            Console.WriteLine("  Config file:      json file that contains the properties and the commands to be run. (Required)");
+            Console.WriteLine("Command: [executor command]|[config command]");
+            Console.WriteLine("  Executor command:   Executor tool commands.");
+            Console.WriteLine("         info       Provides information about the executor tool.");
+            Console.WriteLine("         help       Provides information about the content of the config file. ");
+            Console.WriteLine("  Config command:   Config file specific commands.");
+        }
+                
+        public static void Main(string[] args)
+        {
+            int argsLength = args.Length;
+            if (argsLength > 0)
```

Comment:
May want to flip this to "if (args.Length == 0) { do blah and exit }", so that you fail fast and a dev can immediately see what the result is rather than searching to the end of the if block.